packages/client/hmi-client/src/page/Home.vue

```diff
@@ -33,56 +33,141 @@
-								<h3>You don't have any shared projects</h3>
-								<p>Shared projects will be displayed on this page</p>
-							</template>
-						</div>
-						<div v-else class="carousel">
-							<div class="chevron-left" @click="scroll('left', $event)">
-								<i class="pi pi-chevron-left" />
+						<section class="list-of-projects">
+							<div v-if="!isLoadingProjects && isEmpty(tab.projects)" class="no-projects">
+								<img src="@assets/svg/seed.svg" alt="" />
+								<template v-if="tab.title === TabTitles.MyProjects">
+									<h3>Welcome to Terarium</h3>
+									<div>
+										Get started by creating a
+										<Button
+											label="new project"
+											class="p-button-text new-project-button"
+											@click="isNewProjectModalVisible = true"
+										/>. Your projects will be displayed on this page.
+									</div>
+								</template>
+								<template v-else-if="tab.title === TabTitles.SharedProjects">
+									<h3>You don't have any shared projects</h3>
+									<p>Shared projects will be displayed on this page</p>
+								</template>
 							</div>
-							<div class="chevron-right" @click="scroll('right', $event)">
-								<i class="pi pi-chevron-right" />
+							<div v-else-if="view === ProjectsView.Cards" class="carousel">
+								<div class="chevron-left" @click="scroll('left', $event)">
+									<i class="pi pi-chevron-left" />
+								</div>
+								<div class="chevron-right" @click="scroll('right', $event)">
+									<i class="pi pi-chevron-right" />
+								</div>
+								<ul v-if="isLoadingProjects">
+									<li v-for="i in [0, 1, 2]" :key="i">
+										<tera-project-card />
+									</li>
+								</ul>
+								<ul v-else>
+									<li v-for="project in tab.projects" :key="project.id">
+										<tera-project-card
+											v-if="project.id"
+											:project="project"
+											:project-menu-items="projectMenuItems"
+											@click="openProject(project.id)"
+											@update-chosen-project-menu="selectedProjectMenu = project"
+										/>
+									</li>
+									<li>
+										<section class="new-project-card" @click="isNewProjectModalVisible = true">
+											<div>
+												<img src="@assets/svg/plus.svg" alt="" />
+											</div>
+											<p>New project</p>
+										</section>
+									</li>
+								</ul>
 							</div>
-							<ul v-if="isLoadingProjects">
-								<li v-for="i in [0, 1, 2]" :key="i">
-									<tera-project-card />
-								</li>
-							</ul>
-							<ul v-else>
-								<li v-for="project in tab.projects" :key="project.id">
-									<tera-project-card
-										v-if="project.id"
-										:project="project"
-										@click="openProject(project.id)"
-										@removed="removeProject"
-									/>
-								</li>
-								<li>
-									<section class="new-project-card" @click="isNewProjectModalVisible = true">
-										<div>
-											<img src="@assets/svg/plus.svg" alt="" />
+							<DataTable
```

Comment:
This has to be its own component, the home.vue is not supposed to be that complicated.